### PR TITLE
text changes

### PR DESCRIPTION
**Pull request by wdwatkins**
For #261 

I've addressed all the boxes in #261 except the turning blue part for 
```
Figure 4: Add "blue" to "Drag a [blue] state over its matching bar", and color the word blue w/ the appropriate blue color
```
I made a few other minor wording changes besides what was pointed out there.

### CODE REVIEW

File: js/rank_states_utils.js [hunk 1]
```diff
@@ -271,7 +271,7 @@ function rankEm(barData) {
     .append('text')
       .classed('rankem-title', true)
       .attr('text-anchor','middle')
-      .text('Drag a state over its matching bar');
+      .text('Drag a blue state over its matching bar');
```
[the remark this answers is not in the document]
Comment (mwernimont) [reply js/rank_states_utils.js]: Could you create a variable with the string blue in it and give it the attribute of the color blue and then concatenate it in?

So 
```
var blue = '<span>blue</span>';

blue.setAttribute('style', 'color: blue;');

.text('Drag a' + blue + 'state over its matching bar');
```
?

[the remark this answers is not in the document]
Comment (mwernimont) [reply js/rank_states_utils.js]: Or am I pipe dreaming here? I'm a little burnt out. lol

[the remark this answers is not in the document]
Comment (aappling-usgs) [reply js/rank_states_utils.js]: Yeah, let's scrap it. I tried adding html directly and it wasn't working. I don't think it's that important, now that we've established that there's no super-simple solution.

[the remark this answers is not in the document]
Comment (aappling-usgs) [reply js/rank_states_utils.js]: Oh gosh, and I just noticed that you (David) had already said in the intro to this PR that you had already tried and not succeeded. Thanks anyway David and Marty for the info here, and sorry for not paying attention to your initial PR comment earlier, David.

File: layout/templates/wu_story.mustache [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <h3>How do we use water in the U.S.?</h3>
       
       <p>
-        The U.S. states and territories used over 322 billion gallons per day in 2015. This would cover the continental U.S. in about 2 inches of water over the course of a year. We use water in different ways to serve many economically and culturally important needs. The national breakdown is as follows:
+        We all depend on water every day, ranging from the water from our faucets, to the food we eat, to much of the electricity we use. The U.S. states and territories used nearly 322 billion gallons of water per day in 2015. This would cover the continental U.S. in about two inches of water over the course of a year. The national breakdown of water withdrawals looks like this:
```
Comment (aappling-usgs) [inline layout/templates/wu_story.mustache]: Going all pro storyteller on us here, eh? I like it :smile:

File: layout/templates/wu_story.mustache [hunk 3]
```diff
@@ -21,19 +21,19 @@
...
+   <p>
+      <i>Each "pie slice" represents a portion of U.S. water withdrawals in 2015.</i>
+   </p>
   </div>
-  
   <div class="story-two">
-  
      <p>
-        Although thermoelectric and irrigation are the most dominant categories nationally, the county breakdown reveals regional variation. Below is a map that shows all categories for each county at one time. While you can still see that irrigation and thermoelectric are the largest, there is a stark contrast between the East, where thermoelectric is dominant, and the West, where irrigation is dominant. See the map below for specific findings:
+        Although thermoelectric and irrigation are the dominant categories nationally, the county breakdown reveals regional variation. In the map below, you can see all categories for each county at once. While you can still see that irrigation and thermoelectric are the largest, there is a striking contrast between the East &mdash; where thermoelectric is dominant &mdash; and the West, where irrigation is dominant.
```
Comment (aappling-usgs) [inline layout/templates/wu_story.mustache]: For the parentheticals, I'd prefer either em dashes for both east and west or commas for both east and west. Right now it's one of each

**Pull request merged**